saga/saga.go

```diff
@@ -148,3 +111,48 @@ func (s Saga) StartCompensatingTask(state *SagaState, taskId string, data []byte
 func (s Saga) EndCompensatingTask(state *SagaState, taskId string, results []byte) (*SagaState, error) {
 	return s.logMessage(state, MakeEndCompTaskMessage(state.sagaId, taskId, results))
 }
+
+/*
+ * Should be called at Saga Creation time.
+ * Returns a Slice of In Progress SagaIds
+ */
+func (s Saga) Startup() ([]string, error) {
```

Comment:
Yup, I'm fine with Startup, and also acknowledge that I am not always right, so if there's something you think is better, I'm sorry for having led you astray.

WDYT is best going forward?

(Feel free to just do what you think is best and then merge, don't need to consider this blocking)